pyproject.toml

```diff
@@ -1,8 +1,32 @@
-[tool.poetry]
-name = "specfem2d_kokkos"
+[build-system]
+requires = ["scikit-build-core>=0.10", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "SPECFEMPP"
 version = "0.1.0"
 description = "Kokkos implementation of SpecFEM2D code"
-authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+authors = [
+  { name = "Rohit R Kakodkar", email = "rohit_kakodkar@princeton.edu" },
+  { name = "Lucas Sawade", email = "lsawade@princeton.edu" },
+  { name = "Congyue Cui", email = "ccui@princeton.edu" }
+]
+requires-python = ">=3.9"
+
+[tool.scikit-build]
+cmake.define.CMAKE_BUILD_TYPE = "Release"
+cmake.define.BUILD_TESTS = "OFF"
+cmake.define.ENABLE_SIMD = "ON"
+cmake.define.ENABLE_ATOMICS_BYPASS = "ON"
+cmake.define.Kokkos_ENABLE_OPENMP = "OFF"
+cmake.define.Kokkos_ARCH_NATIVE = "ON"
```

Comment:
For distro, we want to build for the lowest possible architecture. So I think the best build recipe is `-D CMAKE_BUILD_TYPE=Release` `-D BUILD_TESTS=ON`. Setting the build tests option on would let use allow users to check their installation